src/components/PhonesCatalog/PhoneDetailsSingle/PhoneDetailsSingle.js

```diff
@@ -62,7 +62,7 @@ const Phone = (props) => {
                      <p className="price-amount"> Device price: <span>{devicePrice}</span> </p>
                      <button className="btn-addcart" onClick={addInCartHandle}>  Add to Cart </button>
                   </div>
-                  {images ? <img src={images[randomNum]} /> : <h3>No Img...</h3>}
+                  {images ? <img src={images[randomNum]} alt="" /> : <h3>No Img...</h3>}
```

Comment:
On line 28
```js
const randomImageSrc = images.length ? images[Math.floor(Math.random() * images.length)] : null;
```
On line 65
```suggestion
                  {randomImageSrc ? <img src={randomImageSrc} alt="" /> : <h3>No Img...</h3>}
```